src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -194,7 +194,7 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     const {config} = this.match;
     const {organization} = this.props;
     const text = this.errorText;
-    const url = `/settings/${organization.slug}/integrations/${config?.provider.key}/${config?.integrationId}/?tab=codeMappings`;
+    const url = `/settings/${organization.slug}/integrations/${config?.provider?.key}/${config?.integrationId}/?tab=codeMappings`;
```

Comment:
@NisanthanNanthakumar Hmm...isn't this going to have some weird behavior if `config.provider` is falsy? That URL is going to be bogus without it. I am wondering if we should modify the [query we use for the endpoint ](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_stacktrace_link.py#L55) to only give us results when we have a provider so we can type `config` with a `provider`. I don't think we want to load any `RepositoryProjectPathConfig` objects in this view if the provider is missing, yea?

Reply:
yeah @NisanthanNanthakumar i remember us talking about this too, basically can just add `organization_integration__isnull=False`